assets/eip-5725/contracts/mocks/ERC20Mock.sol

```diff
@@ -1,17 +1,12 @@
-// SPDX-License-Identifier: CC0-1.0
+// SPDX-License-Identifier: GPL-3.0
```

Comment:
Assets included with your proposal should be CC0-1.0, and must impose minimal restrictions on downstream projects. It isn't much of a reference implementation if everyone who looks at it is bound to use the GPL.